cmd/schema/schema_test.go

```diff
@@ -45,7 +45,7 @@ var _ = Describe("Schema", func() {
 		Expect(json.Unmarshal([]byte(contents), &s)).To(Succeed())
 		Expect(s.Version).To(Equal("http://json-schema.org/draft-07/schema#"))
 		Expect(s.Definition).ToNot(BeNil())
-		Expect(s.Definitions).To(HaveLen(9))
+		Expect(s.Definitions).To(HaveLen(10))
```

Comment:
@Callisto13 I don't quite get why this change of type caused the list to grow by 1... but I think its fine? :see_no_evil:

Reply:
lol 🤷‍♀️

Reply:
oh wait i think i know 🤔  can you update `cmd/schema/main.go` to remove the new thing whatever it is